runtime/doc/options.txt

```diff
@@ -4949,6 +4949,15 @@ A jump table for the options with a short description can be found at |Q_op|.
 	be used as the new value for 'scroll'.  Reset to half the window
 	height with ":set scroll=0".
 
+			*'scrollback'* *'scbk'* *'noscrollback'* *'noscbk'*
+'scrollback' 'scbk'	boolean  (default: 1000
```

Comment:
This isn't a boolean option, so the "boolean" and "noscrollback"/"noscbk" parts should be removed.